src/modules/producer.js

```diff
@@ -16,10 +15,6 @@ class Producer {
     this.channel = null;
   }
 
-  get connection() {
-    return this._connection;
-  }
-
   set connection(value) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Provide a getter matching this setter for 'connection'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.dialonce.io/coding_rules#rule_key=javascript%3AS2376)